app/views/notify-token-drawer/notify-token-flow.tsx

```diff
@@ -0,0 +1,46 @@
+import { useCallback, useEffect, useRef, useState } from 'react'
+import type { NotifyTokenFormInput } from './notify-token-form'
+import { NotifyTokenForm } from './notify-token-form'
+import type { NotifyEmailFormInput } from './notify-email-form'
+import { NotifyEmailForm } from './notify-email-form'
+import { notifyEmail, notifyToken } from '@/api/notify-token/notify-token'
+
+export interface LinkRecoveryEmailFlowProps {
+  close: () => void
+}
+
+export function NotifyTokenFlow({ close }: LinkRecoveryEmailFlowProps) {
+  const [step, setStep] = useState(1)
+  const [token, setToken] = useState('')
+  const tokenRef = useRef(token)
```

Comment:
I had to send token together with email on second request becasue token is required field.
We can bypass it, adding another required field, like deviceId.